sys/arvo.hoon

```diff
@@ -282,19 +319,20 @@
           ==
       ^-  (unit (unit (cask)))
       ::  ~&  [%arvo-scry ren bed]
-      =+  ^=  old
-          :*  fur
-              ren
-              [%& p.bed]
-              q.bed
-              `coin`[%$ r.bed]
-              (flop s.bed)
-          ==
+      =/  old=scry-sample
+        :*  fur
+            ren
+            [%& p.bed]
+            q.bed
+            `coin`[%$ r.bed]
+            (flop s.bed)
+        ==
       ^-  (unit (unit (cask)))
-      =+  pro=(slym (slap rig [%limb %scry]) old)
+      =+  fun=-:(~(slap wa worm.vane) rig [%limb %scry])
+      =+  pro=-:(~(slym wa worm.vane) fun old)
       ?~  q.pro  ~
       ?~  +.q.pro  [~ ~]
-      =+  dat=(slot 7 pro)
+      =+  dat=-:(~(slot wa worm.vane) 7 pro)
```

Comment:
The type in the `dat` vase doesn't get used at all. We should replace this and the next line with:
```
=/  dat  +>.q.pro
[~ ~ (mark -.dat) +.dat]
```